Documentation/capabilities-guide.md

```diff
@@ -300,6 +300,10 @@ without changing the executed images.
 The `--cap-retain` option to `rkt run` manipulates the `retain` capabilities set.
 The `--cap-remove` option manipulates the `remove` set.
 
+Capabilities specified from the command-line will replace all capability settings in the image manifest.
```

Comment:
Yes. What happens if the manifest specifies _os/linux/capabilities-remove-set_ and I use `--cap-retain` on the command line?